app/script/view_model/ConversationInputEmojiViewModel.js

```diff
@@ -145,6 +145,9 @@ z.ViewModel.ConversationInputEmojiViewModel = class ConversationInputEmojiViewMo
       });
 
     this.bound_remove_emoji_list = this.remove_emoji_popup.bind(this);
+
+    this.should_replace_inline_emoji = ko.observable(properties_repository.get_preference(z.properties.PROPERTIES_TYPE.EMOJI));
```

Comment:
it doesn't really have to be an observable at the moment, nobody is listening to it... I'd just keep it a simple boolean variable.